Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1770,8 +1770,10 @@ static CGImageRef __CGContextCreateRenderableImage(CGImageRef image) {
 #pragma region Drawing Parameters - Stroke / Fill Patterns
 
 template <typename ContextStageLambda> // Takes the form HRESULT(*)(CGContextRef)
-static HRESULT _CreatePatternBrush(
-    CGContextRef context, CGPatternRef pattern, const CGFloat* components, ID2D1BitmapBrush1** brush, ContextStageLambda&& contextStage) {
+static HRESULT _CreatePatternBrush(CGContextRef context,
+                                   CGPatternRef pattern,
+                                   ID2D1BitmapBrush1** brush,
+                                   ContextStageLambda&& contextStage) {
     // TODO #1592: change to support grayscale (masks) after dustins change.
```

Comment:
i have one more commit on top of this, looks like i didn't push it.
It takes the comment out and creates the bitmap based on colorspace.